snappy/dpu-decompress/dpu_task.c

```diff
@@ -13,13 +13,21 @@ __host uint32_t output_offset[NR_TASKLETS];
 __host __mram_ptr uint8_t *input_buffer;
 __host __mram_ptr uint8_t *output_buffer;
 
+// profiling information
+__host uint64_t metrics_tasklets_start[NR_TASKLETS];
+__host uint64_t metrics_tasklets_stop[NR_TASKLETS];
+__host bool metrics_tasklets_did_work[NR_TASKLETS];
+__host perfcounter_config_t metrics_perfctr_config;
+
 int main()
 {
 	struct in_buffer_context input;
 	struct out_buffer_context output;
+	/* perfcounter_config(COUNT_CYCLES, true); */
+	perfcounter_config(metrics_perfctr_config, true);
 	uint8_t idx = me();
 
-	printf("DPU starting, tasklet %d\n", idx);
+	/* printf("DPU starting, tasklet %d\n", idx); */
```

Comment:
I would just remove these commented out print statements. I've never been a fan of keeping commented out code around just in case we need it later.